inference/nlp/realtime/llm/bloom_176b/djl_deepspeed_deploy.ipynb

```diff
@@ -0,0 +1,613 @@
+{
```

Comment:
<p>The notebook doesn't really talk about or show off DJLServing. At least, not that I can see. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>we are wanting to focus on DeepSpeed and Large Language models and the DeepSpeed Container</p><p> </p><p>the idea was not to draw too much attention to DJL</p><p> </p><p>and hence we left it out</p><p> </p>

Reply:
<p>added some helpful discription for DJL and deepSpeed</p>